web-console/src/druid-models/metric-spec.tsx

```diff
@@ -67,14 +67,17 @@ export const METRIC_SPEC_FIELDS: Field<MetricSpec>[] = [
         group: 'max',
         suggestions: ['longMax', 'doubleMax', 'floatMax'],
       },
-      {
-        group: 'first',
-        suggestions: ['longFirst', 'doubleFirst', 'floatFirst'],
-      },
-      {
-        group: 'last',
-        suggestions: ['longLast', 'doubleLast', 'floatLast'],
-      },
+      // Do not show first and last aggregators as they can not be used in ingestion specs and this definition is only used in the data loader.
+      // Ref: https://druid.apache.org/docs/latest/querying/aggregations.html#first--last-aggregator
+      // Should the first / last aggregators become usable at ingestion time, comment in the lines below:
+      // {
+      //   group: 'first',
+      //   suggestions: ['longFirst', 'doubleFirst', 'floatFirst'],
+      // },
+      // {
+      //   group: 'last',
+      //   suggestions: ['longLast', 'doubleLast', 'floatLast'],
+      // },
```

Comment:
We don't usually allow commented out code in java code. Do you think it's better to be consistent in the web-console as well?

Reply:
fixed.